source/locale/pt_BR/LC_MESSAGES/mosca-tls.po

```diff
@@ -0,0 +1,490 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2020, CPqD
+# This file is distributed under the same license as the dojot package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: dojot 0.4.3\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-08-19 16:20-0300\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.8.0\n"
+
+#: ../../source/mosca-tls.rst:4
+msgid "Using MQTT with security (TLS)"
+msgstr "Usando MQTT com segurança (TLS)"
+
+#: ../../source/mosca-tls.rst:7
+msgid "Audience: developers"
+msgstr "Público: desenvolvedores"
+
+#: ../../source/mosca-tls.rst:8
+msgid "Level: intermediate"
+msgstr "Nível: intermediário"
+
+#: ../../source/mosca-tls.rst:9
+msgid "Reading time: 15 m"
+msgstr "Tempo de leitura: 15 ms"
+
+#: ../../source/mosca-tls.rst:12
+msgid ""
+"This document describes how to configure dojot to use MQTT over TLS when "
+"using the microservice `IotAgent-Mosca`_"
+msgstr ""
+"Este documento descreve como configurar o dojot para usar o MQTT sobre "
+"TLS quandousando o microsserviço `IotAgent-Mosca`_ "
+
+#: ../../source/mosca-tls.rst:16
+msgid "Table of Contents"
+msgstr "Índice"
+
+#: ../../source/mosca-tls.rst:18
+msgid "For a device to connect using TLS with Mosca, it must possess:"
+msgstr ""
+"Para que um dispositivo se conecte usando o TLS com o Mosca, ele deve "
+"possuir:"
+
+#: ../../source/mosca-tls.rst:20
+msgid "A key pair (.key file);"
+msgstr "Um par de chaves (arquivo .key)"
+
+#: ../../source/mosca-tls.rst:21
+msgid ""
+"A certificate signed by a Certificate Authority (CA) trusted by Mosca "
+"(.crt file);"
+msgstr ""
+"Um certificado assinado por uma Autoridade de Certificação (CA) confiável"
+" por Mosca(arquivo .crt);"
+
+#: ../../source/mosca-tls.rst:23
+msgid "The certificate of this CA (.crt file);"
+msgstr "O certificado desta CA (arquivo .crt);"
+
+#: ../../source/mosca-tls.rst:26
+msgid ""
+"When a device is created, DeviceManager will automatically notify the "
+"following components:"
+msgstr ""
+"Quando um dispositivo é criado, o DeviceManager notifica automaticamente "
+"oseguintes componentes:"
```

Comment:
> **os seguintes** componentes
Is this translation automatic? There are several sections where there is no space separating the words...